tests/unity/CMakeLists.txt

```diff
@@ -166,6 +166,6 @@ function(cmock_handle header_file)
 
   cmock_linker_wrap_trick(${mod_header_path})
 
-  target_include_directories(app PRIVATE ${CMOCK_PRODUCTS_DIR})
+  target_include_directories(zephyr_interface BEFORE INTERFACE ${CMOCK_PRODUCTS_DIR})
```

Comment:
does zephyr_include_directories(${CMOCK_PRODUCTS_DIR}) work? Or do you need the BEFORE due to aliasing issues?

Reply:
i need before. t that's the trick with mocking inline functions. we create header where inlines are replaced with standard declarations and we need to enforce that it will be included earlier than original header (otherwise it will use static inline implementation).

Reply:
If this causes problems we could maybe look into removing the orignal header's include path, but I see no problems with this approach.